### PR TITLE
CAW21-82 Build accordion paragraph component

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Accordion paragraph component
- https://github.com/SU-SWS/jumpstart_ui/pull/111
- @cjwest please have a quick accessibility review of this please.

# Need Review By (Date)
- 8/10

# Urgency
- medium/high

# Steps to Test
1. checkout this branch and the branch in https://github.com/SU-SWS/jumpstart_ui/pull/111
2. `drush cim -y; drush cr`
3. create a basic page and add an accordion to it
4. save and verify it functions as expected.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by imonroe — APPROVED** [review]
This functions correctly for me.  The styles look good, and there's tests.  It only allows me to make a single entry in the accordion; e.g., it works more like a collapsible panel than an accordion with multiple sections.  The other thing I note is that `minimal-html` is the only text format it will allow in the body of the panel.  If that is the desired functionality, then it seems to be in order.

**Comment by imonroe** [comment]
Holding off on merging until @cjwest gets a chance to review.

**Comment by pookmish** [comment]
for the text format, i figured it's better to start restricted and see how people react to that. Accordion's aren't great for content delivery so i prefer to present some hurdles for them.

**Comment by imonroe** [comment]
> for the text format, i figured it's better to start restricted and see how people react to that. Accordion's aren't great for content delivery so i prefer to present some hurdles for them.

that makes sense to me.  I'm not a huge fan of accordions either.

**Comment by cjwest** [comment]
@imonroe, if possible, can we put this on a dev site for review?

**Comment by pookmish** [comment]
@cjwest https://pr241-q2z6kgjdviexdrpo1sukusqxw8aq6hmc.tugboat.qa/

**Comment by cjwest** [comment]
Thank you, Mike!

One question: How do I log in?
________________________________

**Comment by pookmish** [comment]
All tugboat sites are `tugboat` for username and password

**Comment by cjwest** [comment]
I left comments in the ticket:
https://stanfordits.atlassian.net/browse/CAW21-82?focusedCommentId=113231

tl;dr:

1. We're getting an HTML error with the use of `<summary>`. How about just use a `<div>` ? or find a fix for the error.
3. I'm going to follow up with SODA as I couldn't get NVDA to work as expected. VO worked great, **so this is not a blocker.**
4. The disclosure triangles are appearing on iPhone/Safari and MacOS/Safari. It's not showing in the Chrome emulator.

Screen Shot 2021-07-29 at 3 48 47 PM

**Comment by cjwest** [comment]
> @cjwest https://pr241-q2z6kgjdviexdrpo1sukusqxw8aq6hmc.tugboat.qa/faqs

Awesome!!

**Comment by pookmish** [comment]
I fixed the validation error and the styling issue. I'm going to mark this as done since there's [another ticket](https://stanfordits.atlassian.net/browse/CAW21-83) for the accessibility.
